CONTRIBUTING.md

```diff
@@ -32,17 +32,17 @@ cargo xtest
 #### Code Style, Hints, and Testing
 
 Refer to our [Coding
-Guidelines](https://developers.libra.org/docs/community/coding-guidelines) for
+Guidelines](https://developers.diem.com/docs/community/coding-guidelines) for
 detailed guidance about how to contribute to the project.
 
 #### Documentation
 
-Libra's developer website is also open source (the code can be found in this
-[repository](https://github.com/libra/libra/developers.libra.org/website/)).  It is built using
+Diem's developer website is also open source (the code can be found in this
+[repository](https://github.com/diem/diem/developers.diem.com/website/)).  It is built using
```

Comment:
Github org isn't decided yet: https://fb.workplace.com/groups/2044217752540140/permalink/2458710494424195/

Reply:
Good spot. I'll change back to use `libra` for this in this pr.